web/source/osk/floatingOskView.ts

```diff
@@ -197,7 +199,11 @@ namespace com.keyman.osk {
 
       var c = util.loadCookie('KeymanWeb_OnScreenKeyboard');
 
-      this._Enabled = util.toNumber(c['visible'], 1) == 1;
+      const displayIfActive = util.toNumber(c['visible'], 1) == 1;
+      if(this.displayIfActive != displayIfActive) {
+        // Setter triggers functions that can go recursive without the conditional check.
+        this.displayIfActive = displayIfActive;
+      }
```

Comment:
I am starting to really dislike property getters and setters, because they look like variables but have side-effects that require comments like this and look like nonsense!

This kind of thing shouldn't be necessary 😢

Reply:
I'll see if I can polish it up to where it no longer is necessary; I definitely agree with you that it's not the best look.

Reply:
Looks like this test can be moved to inside the setter with little issue.  That's probably a more intuitive way to manage the property's related behaviors.